File: src/main/java/co/edu/upb/trenes/models/equipaje/NodoEquipaje.java
package co.edu.upb.trenes.models.equipaje;

public class NodoEquipaje {
    private Equipaje equipaje;
    private NodoEquipaje siguiente;

    public NodoEquipaje(Equipaje equipaje) {
        this.equipaje = equipaje;
    }

    public Equipaje getEquipaje() { return equipaje; }
    public NodoEquipaje getSiguiente() { return siguiente; }
    public void setSiguiente(NodoEquipaje siguiente) { this.siguiente = siguiente; }
}

File: docs/legacy-csharp/Gestion de Equipaje y Abordaje/Nodo_Equipaje.cs
public class Nodo_Equipaje 
{

    private Equipaje equipaje;
    private Nodo_Equipaje siguiente;

    public Equipaje Equipaje { get => equipaje; set => equipaje = value; }
    public Nodo_Equipaje Siguiente { get => siguiente; set => siguiente = value; }

    public Nodo_Equipaje(Equipaje equipaje) {

        Equipaje = equipaje;
        Siguiente = null;
    }
}